example/ck_tile/12_smoothquant/instances/smoothquant_bf16_n256_instance.cpp

```diff
@@ -0,0 +1,12 @@
+
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2024, Advanced Micro Devices, Inc. All rights reserved.
+
+#include "smoothquant_instance_common.hpp"
+
+// clang-format off
+//                                                   rm rn tm  tn   vn   pd    2p
```

Comment:
Are the titles not aligned?

Reply:
cb53ad1dac7c32bdee8ff9938fd0f55c3b61d2d2